### DISCUSSION TITLE
Need to enable forgot password on login page

### DISCUSSION THREAD

**Discussion by pranavraneinfovista** [OpenIdentityPlatform/OpenAM#548] [Q&A]
Hi,

We recently upgraded openAM 12 to 14.6.6. We are using the war file running on Tomcat server. The upgrade was error free.

But now, we are not seeing 'Forgot Password' option on landing page. How do we enable that using openAM settings? The usual advice from Google searches doesn't seem to be working.

Thanks in advance.

**Comment by maximthomas** [comment]
Hi @pranavraneinfovista ,
Уou need to customize the login page template. The way of customization depends on UI type you use - XUI or legacy UI
